src/provider/application.js

```diff
@@ -30,9 +30,12 @@ class Application extends EventEmitter {
     this.verifyChallenge = this.verifyChallenge.bind(this);
     this.emitError = this.emitError.bind(this);
     this.unload = this.unload.bind(this);
+    const self = this;
 
-    // Resize for slow loading images
-    document.body.addEventListener('load', this.imageRequestResize.bind(this), true);
+    // Resize for slow loading images when DOM is ready
+    document.addEventListener('DOMContentLoaded', () => {
+      document.body.addEventListener('load', self.imageRequestResize.bind(self), true);
```

Comment:
in `imageRequestResize` will the `target` ever be an image? I'm not sure what usefulness [this will have](https://github.com/cerner/xfc/blob/11060b7cfb2293aeeeffee3706e69f932ea18fbd/src/provider/application.js#L90) if not? I'm also a little lost on how this currently operates as well though...

Reply:
Yes, this was added as part of previous [PR](https://github.com/cerner/xfc/pull/36) where we were seeing an issue while resizing when large images are present.